Config.cmake.in

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
```

Comment:
You should add `check_required_components`. In this case if user wrongly call `find_package(getopt REQUIRED nonexistent_component)` the error will be reported. See https://github.com/forexample/package-example/blob/cf2ea1d6a209fb9eca2ab83fdd0ac15fe4d3e807/Foo/cmake/Config.cmake.in